recipe/meta.yaml

```diff
@@ -11,8 +11,11 @@ source:
   sha256: 84fae3617b70b1219d99a53888d6c404e0e0bc251d13d35dbb84fbf83cf5f894
 
 build:
-  number: 0
-  skip: true  # [py<37]
+  number: {{ build }}
+  noarch: python
+  # avoid collision on build string
+  # with multiple noarch builds
+  string: "py_{{ noarch_platform }}_{{ build }}"
```

Comment:
this is where the sneaky `build` math comes in, e.g. `build * 2` and `(build * 2) + 1`. maybe saw that on `dataclasses` first?

https://github.com/conda-forge/dataclasses-feedstock/blob/main/recipe/meta.yaml

Reply:
Do you think mismatched build numbers is better than specifying the build string?

Reply:
Conda-Build generates a hash based on a package's dependencies. So this should already be unique between the two builds

Reply:
I may have added this when I was trying a solution with multiple outputs where it produced incorrect build strings in combination with noarch. May not be needed!